src/test/java/com/coreos/jetcd/resolver/URIResolversTest.java

```diff
@@ -0,0 +1,35 @@
+package com.coreos.jetcd.resolver;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.URI;
+import org.junit.Test;
+
+public class URIResolversTest {
+  private static final String[] DIRECT_SCHEMES = new String[] { "http", "https" };
+  private static final String[] DNSSRV_SCHEMES = new String[] { "dns+srv", "dnssrv", "srv" };
+
+  @Test
+  public void testDirectResolver() throws Exception {
+    URIResolvers.Direct discovery = new URIResolvers.Direct();
+
+    for (String scheme : DIRECT_SCHEMES) {
+      URI uri = URI.create(scheme + "://127.0.0.1:2379");
+
+      assertThat(discovery.supports(uri)).isTrue();
+      assertThat(discovery.resolve(uri).size()).isGreaterThan(0);
+    }
+  }
+
```

Comment:
Can we add tests for invalid urls for Direct Resolvers.

transform those deleted tests to here.

```java
-  @Test(dataProvider = "invalidEndpoints", expectedExceptions = IllegalArgumentException.class)		
 -  public void testBuild_WithInValidFormatEndpoints(String endpoint) {		
 -    builder.setEndpoints(endpoint);		
 -  }		
 -		
 -  @DataProvider		
 -  private Object[][] invalidEndpoints() {		
 -    return new Object[][] {		
 -        {"://127.0.0.1:2379"},		
 -        {"mailto://127.0.0.1:2379"},		
 -        {"http://127.0.0.1"},		
 -        {"http://127.0.0.1:2379/path"}		
 -    };		
 -  
```